test/serializer.spec.ts

```diff
@@ -6,8 +6,10 @@ import { Parent } from '../src/decorator/parent';
 @Parent({
     discriminatorField: 'type'
 })
-abstract class ParentExample {
-    abstract test(): string;
+class ParentExample {
```

Comment:
the abstract keyword was here to ensure that it could work with abstract parent classes. I think we should have a test that involves abstract parent class along with this one.